core/tests/ts-integration/src/context-owner.ts

```diff
@@ -354,34 +367,33 @@ export class TestContextOwner {
         if (!l2ETHAmountToDeposit.isZero()) {
             // Given that we've already sent a number of transactions,
             // we have to correctly send nonce.
-            const depositHandle = this.mainSyncWallet
-                .deposit({
-                    token: zksync.utils.ETH_ADDRESS,
-                    approveBaseERC20: true,
-                    approveERC20: true,
-                    amount: l2ETHAmountToDeposit as BigNumberish,
-                    approveBaseOverrides: {
-                        nonce: nonce,
-                        gasPrice
-                    },
-                    overrides: {
-                        nonce: nonce + (ethIsBaseToken ? 0 : 1), // if eth is base token the approve tx does not happen
-                        gasPrice
-                    }
-                })
-                .then((tx) => {
-                    const amount = ethers.utils.formatEther(l2ETHAmountToDeposit);
-                    this.reporter.debug(`Sent ETH deposit. Nonce ${tx.nonce}, amount: ${amount}, hash: ${tx.hash}`);
-                    tx.wait();
-                });
-            nonce = nonce + 1 + (ethIsBaseToken ? 0 : 1);
-            this.reporter.debug(
-                `Nonce changed by ${1 + (ethIsBaseToken ? 0 : 1)} for ETH deposit, new nonce: ${nonce}`
-            );
-
-            // Add this promise to the list of L1 tx promises.
-            // l1TxPromises.push(depositHandle);
-            await depositHandle;
+            // const depositHandle = this.mainSyncWallet
+            //     .deposit({
+            //         token: zksync.utils.ETH_ADDRESS,
+            //         approveBaseERC20: true,
+            //         approveERC20: true,
+            //         amount: l2ETHAmountToDeposit as BigNumberish,
+            //         approveBaseOverrides: {
+            //             nonce: nonce,
+            //             gasPrice
+            //         },
```

Comment:
Does this deposit break the custom base token test flow? 
If it does, we probably need another flag to switch it off while testing networks with custom base token.
If not, we probably should uncomment this

Reply:
It breaks on the first `zk test i server` ran, but not in the next ones. If I'm not mistaken, this issue is present on `kl-factory` too, so we should try to see if it can be fixed on a separate PR, the better option might be to uncomment this for this PR so it doesn't make changes to `kl-factory`.